docs/quick-start/configure-mongodb.txt

```diff
@@ -41,15 +41,20 @@ Configure Your MongoDB Connection
 
          // ...
 
-   .. step:: Add the Laravel MongoDB provider
+   .. step:: Add the MongoDB provider
 
-      Open the ``app.php`` file in the ``config`` directory and
-      add the following entry into the ``providers`` array:
+      Open the ``providers.php`` file in the ``bootstrap`` directory and add
```

Comment:
Q: is this the correct file name? I only see an `app.php` file and a `cache` folder in the bootstrap directory

Reply:
Thanks for checking. 
Yes, this is the correct file and path. I think you might be looking at Laravel 10.x, but this is for 11.x. The Laravel 11.x docs refer to this file here:
https://laravel.com/docs/11.x/providers#registering-providers